docs/fundamentals/code-analysis/quality-rules/ca2253.md

```diff
@@ -31,6 +31,8 @@ Named placeholders in the logging message template should not be comprised of on
 
 Rename the numeric placeholder.
 
+See the [LoggerExtensions.LogInformation Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions.loginformation) for usage examples.
```

Comment:
```suggestion
See the [LoggerExtensions.LogInformation Method](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.loggerextensions.loginformation) for usage examples.
```

Remove the locale from the URL, but more importantly - I'm curious about the text in this sentence as this specific API ref doesn't have any example usages.

Reply:
Hi @IEvangelist I see in that page few examples:


logger.LogInformation(exception, "Error while processing request from {Address}", address)
logger.LogInformation(0, exception, "Error while processing request from {Address}", address)

etc